updated docs to rename the binary per the os system you are on (#35)
CrunchyData/postgres-operator-client: docs/content/_index.md

## Installing the Client

### Linux or Mac

> If you downloaded the Linux binary rename the binary from `kubectl-pgo-linux-amd64` to `kubectl-pgo` or if you download the Mac binary rename the binary from `kubectl-pgo-darwin-amd64` to `kubectl-pgo` then move the `kubectl-pgo` binary to `/usr/local/bin` and make it executable by running the following commands:

Reviewer: :pick:

```suggestion
If you downloaded the Linux binary, rename the binary from `kubectl-pgo-linux-amd64` to `kubectl-pgo`. If you downloaded the Mac binary, rename the binary from `kubectl-pgo-darwin-amd64` to `kubectl-pgo`. Once renamed, move the `kubectl-pgo` binary to `/usr/local/bin`, and make it executable by running the following commands:
```

```
sudo mv /PATH/TO/kubectl-pgo /usr/local/bin/kubectl-pgo
sudo chmod +x /usr/local/bin/kubectl-pgo
```

### Windows

> If you downloaded the Windows binary rename `kubectl-pgo-windows-386` to `kubectl-pgo.exe` this allows Windows to recognize the file as an executable.  From there open a power shell terminal in the same directory to execute the binary.

Reviewer: :pick: 
```suggestion
If you downloaded the Windows binary, rename `kubectl-pgo-windows-386` to `kubectl-pgo.exe`. This allows Windows to recognize the file as an executable.  From there, open a power shell terminal in the same directory to execute the binary.
```

:thinking: Is opening a terminal necessary? Are we prompting the user to just leave the terminal open or to actually execute the binary?

Author: It was in the notes in the story of what a customer had to do.  I can remove it, I'm not on windows so I wasn't sure if it was needed or not
